server/lib/host-metrics.js

```diff
@@ -258,6 +258,29 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(host, { startDate, endDate, timeUnit } = {}) {
+  const hostedCollectives = await host.getHostedCollectives();
+  const ids = hostedCollectives.map(c => c.id);
+
+  const results = await sequelize.query(
+    `SELECT t1."netAmountInCollectiveCurrency" as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
+FROM "Transactions" as t1
+WHERE t1."CollectiveId" IN (${ids})
```

Comment:
Variables need to be injected using `replacements`

Reply:
Done. 👍🏽